docs/annotating_code/supported_annotations.md

```diff
@@ -18,6 +18,8 @@ Psalm uses the following PHPDoc tags to understand your code:
   Used to specify what properties can be read on object that uses `__get`
 - [`@property-write`](https://docs.phpdoc.org/references/phpdoc/tags/property-write.html)
   Used to specify what properties can be written on object that uses `__set`
+- [`@method`](https://docs.phpdoc.org/latest/references/phpdoc/tags/method.html)
```

Comment:
Could you also fix the other links within this paragraph which do not use the correct deeplinks, e.g. https://docs.phpdoc.org/latest/references/phpdoc/tags/property-read.html

Reply:
Sure